src/jspdf.js

```diff
@@ -2939,7 +2939,7 @@ var jsPDF = (function(global) {
      *
      * If `type` argument is undefined, output is raw body of resulting PDF returned as a string.
      *
-     * @param {string} type A string identifying one of the possible output types. Possible values are 'arraybuffer', 'blob', 'bloburi'/'bloburl', 'datauristring'/'dataurlstring', 'datauri'/'dataurl', 'dataurl
+     * @param {string} type A string identifying one of the possible output types. Possible values are 'arraybuffer', 'blob', 'bloburi'/'bloburl', 'datauristring'/'dataurlstring', 'datauri'/'dataurl', 'dataurlnewwindow', 'pdfobjectnewwindow', 'pdfjsnewwindow'.
      ', 'pdfobjectnewwindow', 'pdfjsnewwindow'.
```

Comment:
sorry for nitpicking, but now `'pdfobjectnewwindow', 'pdfjsnewwindow'` appear twice ;)

Reply:
Ah, thanks for catching that!